lib/kaitai/kaitai/kaitaistream.h

```diff
@@ -162,7 +167,11 @@ class kstream {
 
     static std::string bytes_strip_right(std::string src, char pad_byte);
     static std::string bytes_terminate(std::string src, char term, bool include);
-    static std::string bytes_to_str(std::string src, std::string src_enc);
+    static std::string bytes_to_str(const std::string src, const char *src_enc);
+    // workaround for https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/issues/67
+    static std::string bytes_to_str(const std::string src, const std::string& enc) {
+      return bytes_to_str(src, enc.c_str());                       
+    }
```

Comment:
only changed I made to the kaitai files except copy-pasting in the newest commit and moving them to match the includes the kaitai-compiler generates.